dl4j-distributed-training-examples/src/main/java/org/deeplearning4j/distributedtrainingexamples/patent/TrainPatentClassifier.java

```diff
@@ -119,7 +119,7 @@
     private int totalExamplesTest = 10000;
 
     @Parameter(names = {"--wordVectorsPath"}, description = "Word vectors path")
-    private String wordVectorsPath = "wasbs://resources@dl4jdata.blob.core.windows.net/wordvectors/GoogleNews-vectors-negative300.bin.gz";
+    private String wordVectorsPath = "wasbs://resources@deeplearning4jblob.blob.core.windows.net/wordvectors/GoogleNews-vectors-negative300.bin.gz";
```

Comment:
this needs to be changed back to 
"wasbs://resources@dl4jdata.blob.core.windows.net/wordvectors/GoogleNews-vectors-negative300.bin.gz";

Reply:
Done